Sources/PlayingCard.swift

```diff
@@ -31,13 +31,13 @@ public func ==(lhs: PlayingCard, rhs: PlayingCard) -> Bool {
 extension PlayingCard: Comparable {}
 
 public func <(lhs: PlayingCard, rhs: PlayingCard) -> Bool {
-    return lhs.suit < rhs.suit || (lhs.suit == rhs.suit && lhs.rank < rhs.rank)
+    return lhs.rank == rhs.rank ? lhs.suit == rhs.suit : lhs.rank < rhs.rank
 }
 
 // MARK: - CustomStringConvertible
 
 extension PlayingCard : CustomStringConvertible {
     public var description: String {
-        return "\(suit)\(rank)"
+        return "\(rank)\(suit)"
```

Comment:
I didn't make sense to me to have the suit first but I noticed that it prints better that way so I reverted it back.